frontend/components/CopyToClipboardButton.vue

```diff
@@ -0,0 +1,36 @@
+<script setup lang="ts">
+import { computed } from 'vue'
+
+const { t } = useI18n()
+
+interface Props {
+  value: string
+  cta: string
+}
+const props = defineProps<Props>()
+
+const prefix = 'CopyToClipboardButton'
+const tt = (key: string) => t(`${prefix}.${key}`)
+
+const statePrefix = `${prefix}[${useStateIDGenerator().id()}]`
```

Comment:
I've always thought our state approach (`useStateIDGenerator`) was pretty heavyhanded, turns out we can just use Vue's `ref` when we don't want it shared across instances, e.g. https://github.com/nuxt/nuxt/issues/15110

This doesn't have an SSR component, but generally when we don't want to share state between components, it's because we're recording user interaction stuff (as here), so there's no SSR component anyway.